src/components/Footer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-footer class="bg-grey-lighten-1" style="max-height: 20%">
+  <v-footer class="bg-grey-lighten-1" style="position: absolute; bottom: -0; width: 100%;overflow: hidden;">
```

Comment:
Is the value "-0" desired?

Reply:
It was supposed to be zero so that the footer sticks to the bottom; I mistakenly included the minus sign.
I will rectify it in the next push. 
Sorry for that!